app/src/main/java/com/example/pictureupload/ui/screens/landing_screen/LandingScreenViewModel.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.pictureupload.ui.screens.landing_screen
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.pictureupload.ui.screens.landing_screen.states.ImageUploadState
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+
+class LandingScreenViewModel: ViewModel() {
+
+  private val _imageState = MutableStateFlow<ImageUploadState>(ImageUploadState.Empty)
+  val imageState: StateFlow<ImageUploadState> = _imageState
+
+  init {
+    initImageState()
+  }
+
+  private fun initImageState() {
```

Comment:
Write some comment in this code to signify that it won't be present in final code